src/components/navigation-accordion/navigation-accordion.js

```diff
@@ -57,9 +57,9 @@ export default class NavigationAccordion extends React.Component {
     return (
       <div
         className={classnames(
-          'px12 flex txt-uppercase txt-fancy round-full w-full color-text',
+          'px12 flex txt-uppercase txt-fancy round-full w-full',
           {
-            'bg-blue-faint color-blue': isActiveSection,
+            'bg-blue-faint color-blue-deep': isActiveSection,
```

Comment:
💡 This slipped by me during NavigationAccordion: the color contrast was not high enough here.

Reply:
![image](https://user-images.githubusercontent.com/19801577/133858286-eefcc744-8f9a-4265-b938-45667e31a9b7.png)

![image](https://user-images.githubusercontent.com/19801577/133858307-930498f7-1f72-4211-afa3-07ca83503708.png)

@katydecorah maybe I'm doing something wrong, but I see `color-text` overriding the `color-blue-deep` here 🤔

Reply:
Wow! Thank you for catching. I'll move `color-text` out of the default classname styles.